roles/sft-server/templates/sftd.vhost.conf.j2

```diff
@@ -1,3 +1,7 @@
+limit_req_zone "$binary_remote_addr$uri" zone=reqs_per_addr:12m rate=10r/s;
+limit_conn_zone "$binary_remote_addr$uri" zone=conns_per_addr:10m;
```

Comment:
Can someone remind me what the uri's will look like during some SFT related call?